resharper/resharper-unity/src/ProjectModel/Caches/UnityProjectFileCacheProvider.cs

```diff
@@ -156,6 +156,9 @@ private static Version TryGetUnityVersionFromDll(XmlElement documentElement)
                 return null;
 
             var filePath = FileSystemPath.Parse(referencePathElement.InnerText);
+            if (!filePath.IsAbsolute)
```

Comment:
If it's not absolute, does that mean it's relative to the project file? Can we use that to try to resolve it and continue?

Reply:
Unfortunately I don't have repro. Just Exception https://youtrack.jetbrains.com/issue/RIDER-21237